app/assets/javascripts/application.js

```diff
@@ -15,4 +15,5 @@
 //= require jquery_ujs
 //= require global
 //= require turbolinks
+//= require action-slider
```

Comment:
Yes i'm using it for my action messages to disappear after 4 seconds of signing in